src/lib/bundler.ts

```diff
@@ -82,7 +82,7 @@ export async function generateNgBundle(ngPkg: NgPackageData): Promise<void> {
 
   // 9. WRITE PACKAGE.JSON and OTHER DOC FILES
   log.info('Writing package metadata');
-  const packageJsonArtifactPaths: NgArtifacts = artifactFactory.calculateArtifactPathsForPackageJson(ngPkg);
+  const packageJsonArtifactPaths = artifactFactory.calculateArtifactPathsForPackageJson(ngPkg);
```

Comment:
Part of the reason for including the explicit typing was so that people could see exactly what was returned from the function without needing an IDE or navigating to the file with the called function. Personally, I find it harder to know what is happening in the code just by reading on github without those explicit typings.